example.php

```diff
@@ -12,12 +12,13 @@
 </head>
 <body>
 <?php
-if(!isset($_SESSION['steamid'])) {
+if(!isset($_SESSION['steamid']) && !isset($_GET['login'])) {
 
     echo "welcome guest! please login \n \n";
     steamlogin(); //login button
     
 }  else {
+    steamlogin(); //login
```

Comment:
This doesnt make sense at all.
Youre displaying a login button even if the user is logged in.